### PR TITLE
add thetaSketch aggregator

### PR DESCRIPTION
**Pull request by chromevoid**

### CODE REVIEW

**Comment by vigith** [comment]
please add testing

**Review by vigith — APPROVED** [review]
LGTM

File: builder/aggregation/thetasketch_test.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestThetaSketch(t *testing.T) {
+	thetaSketch := NewThetaSketch()
+	thetaSketch.SetName("output_name").SetFieldName("metric_name").SetIsInputThetaSketch(false).SetSize(16384)
+
+	// "omitempty" will ignore boolen=false
```
Comment (jbguerraz) [inline builder/aggregation/thetasketch_test.go:13]: typo, boolen => boolean

**Review by jbguerraz — CHANGES_REQUESTED** [review]

**Review by jbguerraz — APPROVED** [review]
Thank you @chromevoid :+1:

**Pull request merged**